.travis.yml

```diff
@@ -16,7 +16,7 @@ matrix:
       env: PYTHON=/usr/bin/python3.4 CATKIN_VERSION=indigo-devel
     - os: osx
       python: 2
-      env: PYTHON=/usr/bin/python CATKIN_VERSION=indigo-devel
+      env: PYTHON=/usr/local/bin/python2 CATKIN_VERSION=indigo-devel
```

Comment:
I gave up trying to get any of this working with system Python on Travis— it was easiest to just focus on brewed python2 and python3, and explicitly reference the exact binaries until we're safely in a virtualenv.